cpp/data/zotero_cgi/qa.html

```diff
@@ -52,6 +52,7 @@ <h1>QA Settings</h1>
                 <p>Add new journal setting:</p>
                 <!-- The quantifier inside the pattern needs to be written as "{{3}" due to Template.h escaping rules! -->
                 <input name="add_tag" placeholder="3-digit tag number" required="required" minlength="3" maxlength="3" pattern="[0-9]{{3}">
+                <input name="add_subfield_code" placeholder="3-character subfield code" required="required" minlength="1" maxlength="1" pattern="[0-9a-z]{{1}">
```

Comment:
copy/paste error in placeholder attribute: should be `1`-character subfield code

Reply:
problem still exists